proto/process/agent.proto

```diff
@@ -436,6 +436,9 @@ message Connection {
 	// DNS-specific metrics
 	uint32 dnsSuccessfulResponses = 25;
 	uint32 dnsFailedResponses = 26;
+	uint32 dnsTimeouts = 27;
+	uint64 dnsSuccessLatency = 28;
+	uint64 dnsFailureLatency = 29;
```

Comment:
Is this meant to be an average or a sum ?

Reply:
Sum. Should I change the name to reflect that?